keyloop/api/v1/exceptions.py

```diff
@@ -1,7 +1,3 @@
-# from pyramid.exceptions import HTTPBadRequest
-# from pyramid.view import exception_view_config
-#
-#
-# @exception_view_config(path_info='/api/v1/', context=Exception)
-# def HTTPErrorHandler(self):
-#     raise HTTPBadRequest("")
+class NotFound(Exception):
```

Comment:
O nome desta classe não condiz com a doc string e parece estar substituíndo a HTTPNotFound.

```suggestion
class IdentityNotFound(Exception):
```

A HTTPNotFound não serve?

Reply:
A gente ta usando para levantar um erro de banco de dados, por isso esse NotFound e não um HTTPNotFound. Mas trocamos o nome para o que você surgeriu.